src/officehours/settings.py

```diff
@@ -72,7 +72,7 @@ def str_to_bool(val):
 
 if DEBUG:
     INSTALLED_APPS += [
-        'drf_yasg',
+        'drf_spectacular',
```

Comment:
`drf_spectacular` was recommended by `drf_yasg` maintainers for OpenAPI 3 schema generation. We've also used it with Canvas App Explorer.

Reply:
Looks like a good switch